collector/zfs.go

```diff
@@ -24,20 +24,16 @@ import (
 	"github.com/prometheus/common/log"
 )
 
-var zfsNotAvailableError = errors.New("ZFS / ZFS statistics are not available")
+var errorZFSNotAvailable = errors.New("ZFS / ZFS statistics are not available")
```

Comment:
We were just talking about this on IRC.  Best practice, not caught by golint as of now. I'll file an issue against golint.